packages/@react-spectrum/s2/src/index.ts

```diff
@@ -72,6 +72,7 @@ export {RangeCalendar, RangeCalendarContext} from './RangeCalendar';
 export {RangeSlider, RangeSliderContext} from './RangeSlider';
 export {SearchField, SearchFieldContext} from './SearchField';
 export {SegmentedControl, SegmentedControlItem, SegmentedControlContext} from './SegmentedControl';
+export {SelectBox, SelectBoxContext, SelectBoxGroup} from './SelectBoxGroup';
```

Comment:
can we remove the SelectBoxContext from being exported? instead, export the SelectBoxGroupContext